test/js/node/stream/node-stream.test.js

```diff
@@ -386,6 +386,27 @@ it("Readable.fromWeb", async () => {
   expect(Buffer.concat(chunks).toString()).toBe("Hello World!\n");
 });
 
+it("Writable.toWeb", async () => {
+  let dataBuffer = "";
+
+  // Create a Node.js Writable stream
+  const nodeWritable = new Writable({
```

Comment:
Where is error handling tested?

Reply:
just wrote some tests for it; errors resolve to `undefined`, so going to fix that now

Reply:
done!